src/pybind/mgr/dashboard/tox.ini

```diff
@@ -35,7 +35,7 @@ basepython=python3
 deps =
     {[base]deps}
     {[base-test]deps}
-    -rrequirements-extra.txt
+   -rrequirements-extra.txt
```

Comment:
this is still appearing in your PR diff. Maybe there is some unintentional space added here? Can you reset the file as it is in main?

`git checkout origin/main src/pybind/mgr/dashboard/tox.ini`